sdk/workloads/Azure.ResourceManager.Workloads/Azure.ResourceManager.Workloads.sln

```diff
@@ -1,19 +1,15 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30309.148
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32526.322
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{4E575FEF-2F10-4B19-9629-9C7EADDF0DC9}") = "Azure.ResourceManager.Workloads", "src\Azure.ResourceManager.Workloads.csproj", "{1B8D2B24-36D1-4B3D-A4A9-4068B6FD9689}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.Workloads", "src\Azure.ResourceManager.Workloads.csproj", "{1B8D2B24-36D1-4B3D-A4A9-4068B6FD9689}"
 EndProject
-Project("{4E575FEF-2F10-4B19-9629-9C7EADDF0DC9}") = "Azure.ResourceManager.Workloads.Tests", "tests\Azure.ResourceManager.Workloads.Tests.csproj", "{515B4C45-425E-4C19-B478-4E87056E296F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.Workloads.Tests", "tests\Azure.ResourceManager.Workloads.Tests.csproj", "{515B4C45-425E-4C19-B478-4E87056E296F}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{92DB83C0-E0FC-4BEA-955A-B3E8ED6FEA6D}"
```

Comment:
I think you can remove Azure.Core.TestFramework, we have add it for all the test project.